xbmc/addons/kodi-addon-dev-kit/CMakeLists.txt

```diff
@@ -2,6 +2,17 @@ project(kodi-addon-dev-kit)
 
 cmake_minimum_required(VERSION 2.8.7)
 
+# add a target to generate API documentation with Doxygen
+find_package(Doxygen)
+
+if(DOXYGEN_FOUND)
+  add_custom_target(doc
+    ${DOXYGEN_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/doxygen/Doxyfile
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+    COMMENT "Generating API documentation with Doxygen" VERBATIM
+  )
+endif(DOXYGEN_FOUND)
+
```

Comment:
Thanks, you mean to have it here, on e.g. AddonModuleXbmcgui.cpp?

```cpp
  PyDoc_STRVAR(XBMCAddon_xbmcgui_ListItem_setArt__doc__,
               "setArt(values) -- Sets the listitem's art \n"
"\n"
"values : dictionary - pairs of { label: value }.\n"
"\n"
"    - Some default art values (any string possible):\n"
"        - thumb : string - image filename\n"
"        - poster : string - image filename\n"
"        - banner : string - image filename\n"
"        - fanart : string - image filename\n"
"        - clearart : string - image filename\n"
"        - clearlogo : string - image filename\n"
"        - landscape : string - image filename\n"
"        - icon : string - image filename\n"
"\n"
"\n"
"\n"
"example:\n"
"        - self.list.getSelectedItem().setArt({ 'poster': 'poster.png', 'banner' : 'banner.png' }) \n");
```

Good to know then I can check it with the new doxygen documentation style.